### PR TITLE
Update ScoreReport's player ID keys

### PR DESCRIPTION
**Pull request by Jackenmen**

### CODE REVIEW

File: pkg/game/elements/score_report.go [hunk 1]
```diff
@@ -1,6 +1,9 @@
 package elements
 
 type ScoreReport struct {
-	ReceivedPoints  map[int]uint32
-	ReturnedMeeples map[int][]uint8
+	// ReceivedPoints[playerID (uint8) = player's received points
```
Comment (Lenetis) [inline pkg/game/elements/score_report.go:4]:
```
// ReceivedPoints[playerID (uint8) 
Missing a closing ']' ------------^
```

**Review by Lenetis — APPROVED** [review]
Everything looks ok, but why are there two getTestScoreReport() functions differing only by first letter case?

File: pkg/game/elements/score_report.go (hunk 1 above)
Comment (Jackenmen) [reply pkg/game/elements/score_report.go]: Done.

**Comment by Jackenmen** [comment]
> but why are there two getTestScoreReport() functions differing only by first letter case?

I forgot to remove it when I moved it into a common test utilities package. It's removed now.

**Review by Lenetis — APPROVED** [review]

**Review by BognaLew — APPROVED** [review]

**Pull request merged**